File: sdkwork-app-sdk-typescript/src/types/announcement-query-form.ts
/** 公告查询Form */
export interface AnnouncementQueryForm {
  /** 公告类型 */
  type?: string;
  /** 公告状态 */
  status?: string;
  /** 每页大小 */
  pageSize?: number;
  /** 页码 */
  pageNum?: number;
}

File: sdkwork-app-sdk-csharp/Models/AnnouncementQueryForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class AnnouncementQueryForm
    {
        public string? Type { get; set; }
        public string? Status { get; set; }
        public int? PageSize { get; set; }
        public int? PageNum { get; set; }
    }
}
